src/configurations/destinations/bq/ui-config.json

```diff
@@ -72,6 +72,16 @@
           "label": "Sync Frequency",
           "value": "syncFrequency",
           "options": [
+            {
+              "name": "Every 5 minutes",
+              "value": "5",
+              "featureFlag": "LOW_GRANULAR_WH_SYNC"
```

Comment:
`HIGH_GRANULARITY_WH_SYNC` . High means with a great level of detail were low means the opposite.